README.md

```diff
@@ -89,19 +89,22 @@ parser.parse(
    c:Tom a c:Cat.
    c:Jerry a c:Mouse;
            c:smarterThan c:Tom.`,
-  (error, quad, prefixes) => {
+  (error, quad) => {
     if (quad)
       console.log(quad);
     else
-      console.log("# That's all, folks!", prefixes);
```

Comment:
When was this hash functionality as part of the callback deprecated?

Reply:
It wasn’t, but I interpreted @RubenVerborgh’s text above as this would be deprecated, but I see I could be wrong. I’ll re-add this!